gordon/main.py

```diff
@@ -83,14 +83,33 @@ def setup(config_root=''):
     return config
 
 
+def _log_or_raise_exceptions(errors, debug):
```

Comment:
Nitpick, but would something like `_log_or_exit_on_exceptions` as the function name be more accurate?